WebApp/handler/AuthontificationHandler.py

```diff
@@ -0,0 +1,72 @@
+from WebAppDIRAC.Lib.WebHandler import WebHandler, asyncGen
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getUsernameForID
+from DIRAC.FrameworkSystem.Client.NotificationClient   import NotificationClient
+from WebAppDIRAC.Lib import Conf
+
+import json
+import tornado
+import requests
+
+class AuthontificationHandler(WebHandler):
```

Comment:
Typo: `Authontification` --> `Authentification` (with `e` and not with `o`) This is also wrong in the filename and other places.